# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLSelectAccessible.h
# Commit: 06263018f1f7
# Full Hash: 06263018f1f70ceb780dff123c581c58c64d24e7
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-11-10 09:24:53
# Regressor Bug: 1733263
# File Overlap Count: 1
# Description:
#   Bug 1733263 - P5: Rely on DOMMenuItem events for ACTIVE state changes in select elements. r=Jamie
#   
#   1. Use dom events in RootAccessible to fire ACTIVE state changes.
#   2. Add DOMMenuItemInactive events to nsListControlFrame and fire it on
#      the previously active option.
# ==============================================================================

diff -r c8f226476a30 -r 06263018f1f7 accessible/html/HTMLSelectAccessible.h
--- a/accessible/html/HTMLSelectAccessible.h	Tue Nov 09 23:17:17 2021 +0000
+++ b/accessible/html/HTMLSelectAccessible.h	Tue Nov 09 23:17:17 2021 +0000
@@ -71,6 +71,8 @@
   virtual nsRect RelativeBounds(nsIFrame** aBoundingFrame) const override;
   virtual void SetSelected(bool aSelect) override;
 
+  nsresult HandleAccEvent(AccEvent* aEvent) override;
+
   // ActionAccessible
   virtual uint8_t ActionCount() const override;
   virtual void ActionNameAt(uint8_t aIndex, nsAString& aName) override;